be/src/exec/vectorized/schema_scanner/schema_helper.cpp

```diff
@@ -78,6 +78,14 @@ Status SchemaHelper::get_table_privs(const std::string& ip, const int32_t port,
             [&request, &result](FrontendServiceConnection& client) { client->getTablePrivs(*result, request); });
 }
 
+Status SchemaHelper::get_tables_meta(const std::string& ip, const int32_t port, const TGetTablesMetaRequest& var_params,
```

Comment:
tables_config is a better name for this table?  

information_schema.tables is also metadata.

Reply:
done